content/en/docs/reference/access-authn-authz/authorization.md

```diff
@@ -337,6 +310,33 @@ A reload **must not** add or remove Node or RBAC authorizers (they can be reorde
 but cannot be added or removed).
 {{< /note >}}
 
+### Command line authorization mode configuration {#using-flags-for-your-authorization-module}
+
+{{< feature-state state="stable" for_k8s_version="v1.8" >}}
```

Comment:
1.8 is a long history ... I think we can safely remove this line.

Reply:
When I added it, I wanted readers to see the contrast between the two choices; one clearly alpha, the other clearly GA.

Reply:
I can remove it if folks feel strongly about it. FWIW, I do see other docs with stable features and really early versions of when it became stable.
Few examples:
https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/#pod-readiness-gate
https://kubernetes.io/docs/reference/setup-tools/kubeadm/implementation-details/

Reply:
Just to be clear, where there's no contrasting ("here's a new thing", "here's a stable thing") then we typically do clean them up.

Reply:
removed